### PR TITLE
filter dns name retrieve from DS

### PR DESCRIPTION
**Pull request by tdegiacinto**
This will filter out dns hostname like .localdomain

### CODE REVIEW

**Review by plarivee — CHANGES_REQUESTED** [review]
Please escape the literal '.' in the domain name of your regex

File: files/install/host-cleaner.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def clean_ds():
 
     # Get all ds computer
     ds_computers = [attr['dNSHostName'][0].lower()
-                    for c_dn, attr in ds.computers if 'dNSHostName' in attr]
+                    for c_dn, attr in ds.computers if 'dNSHostName' in attr and re.match('.*cloud.coveo.com$', attr['dNSHostName'][0])]
```
Comment (plarivee) [inline files/install/host-cleaner.py]: Escape the literal '.' in the regex

Comment (tdegiacinto) [reply files/install/host-cleaner.py]: Nope it's wanted, it's part of the regex expression

**Review by mouellet-coveo — APPROVED** [review]

File: files/install/host-cleaner.py (hunk 1 above)
Comment (plarivee) [reply files/install/host-cleaner.py]: `'.*\.cloud\.coveo\.com$' ` else  somethingsomethingBcoveoRcloudQcom would match.

Comment (tdegiacinto) [reply files/install/host-cleaner.py]: Done

**Review by plarivee — APPROVED** [review]

**Pull request merged**